src/test/kotlin/coverosR3z/server/SocketTests.kt

```diff
@@ -0,0 +1,103 @@
+package coverosR3z.server
+
+import coverosR3z.logging.logInfo
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.net.ServerSocket
+import java.net.Socket
+
+
+class ClientHandler(): Runnable {
```

Comment:
It is probably appropriate to separate socket logic out into a non tests package